ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -228,13 +233,17 @@ import { focusOnFirstError } from '@/utils'
 import { MhrRegistrationHomeOwnersIF } from '@/interfaces/mhr-registration-interfaces'
 import { SimpleHelpToggle } from '@/components/common'
 /* eslint-enable no-unused-vars */
+import HomeOwnerGroups from '@/components/mhrRegistration/HomeOwners/HomeOwnerGroups.vue'
```

Comment:
Might be able to simplify import path: `./HomeOwnerGroups.vue`
super minor

Reply:
Done.